bin/sce_to_anndata.R

```diff
@@ -16,9 +16,20 @@ option_list <- list(
     help = "path to rds file with input sce object to be converted"
   ),
   make_option(
-    opt_str = c("-o", "--output_h5_file"),
+    opt_str = c("--output_rna_h5"),
     type = "character",
-    help = "path to output hdf5 file to store AnnData object. Must end in .hdf5 or .h5"
+    help = "path to output hdf5 file to store RNA counts as AnnData object. Must end in .hdf5 or .h5"
+  ),
+  make_option(
+    opt_str = c("--feature_name"),
+    type = "character",
+    help = "Feature type. Must match the altExp name, if present."
+  ),
+  make_option(
+    opt_str = c("--output_feat_h5"),
```

Comment:
Three extra characters seem worth it here
```suggestion
    opt_str = c("--output_feature_h5"),
```